composer.json

```diff
@@ -1,10 +1,75 @@
 {
-  "name": "su-sws/stanford_publication",
-  "description": "A Drupal feature.",
-  "type": "drupal-custom-module",
-  "license": "GPL-2.0+",
-  "minimum-stability": "dev",
-  "require": {
-      "composer/installers": "^1.2"
-  }
+    "name": "su-sws/stanford_publication",
+    "description": "A Drupal publication content type and views.",
+    "type": "drupal-custom-module",
+    "license": "GPL-2.0-or-later",
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "repositories": [
+        {
+            "type": "composer",
+            "url": "https://packages.drupal.org/8"
+        },
+        {
+            "type": "package",
+            "package": {
+                "name": "citation-style-language/locales",
+                "version": "1.0.0",
+                "source": {
+                    "type": "git",
+                    "url": "https://github.com/citation-style-language/locales.git",
+                    "reference": "master"
+                }
+            }
+        },
+        {
+            "type": "package",
+            "package": {
+                "name": "citation-style-language/styles-distribution",
+                "version": "1.0.0",
+                "source": {
+                    "type": "git",
+                    "url": "https://github.com/citation-style-language/styles-distribution.git",
+                    "reference": "master"
+                }
+            }
+        }
+    ],
+    "require": {
+        "citation-style-language/locales": "1.0.0",
+        "citation-style-language/styles-distribution": "1.0.0",
+        "composer/installers": "^1.2",
+        "drupal/element_class_formatter": "^1.3",
+        "drupal/field_formatter_class": "^1.0",
+        "drupal/field_group": "^3.0",
```

Comment:
I remember this module being pretty unstable and causing issues oh, probably 2 years ago now, (Earth). Any concerns or research on adding it now?

Reply:
We require `field_group` in stanford_news, `stanford_person` and the profile so i'm confident it works well these days.